### PR TITLE
[LOCATION-18] Show form when countries is 1

### PR DESCRIPTION
**Pull request by Arturo777**
### What problem is solving?

When the store only has one country shipping, the `dn` class (display none) is activated and hide the country selector, but the form should shown automatically, and it was not showing because no country was selected.

### Solution
If the store has only one country, select by default that country and it will trigger to show the form fields.

| Only shipments to | Before | After |
|--------|-------|--------|
| USA | [Workspace](https://before--eriksbikeshop.myvtex.com/)    | [Workspace](https://dnclass--eriksbikeshop.myvtex.com/)   |
| MEX | [Workspace](https://before--farmaciasklyns.myvtex.com/)    | [Workspace](https://arturodev--farmaciasklyns.myvtex.com/)   |

### CODE REVIEW

File: react/components/LocationForm.tsx [hunk 1]
```diff
@@ -580,25 +583,34 @@ const LocationForm: FunctionComponent<WrappedComponentProps & AddressProps> = (
     }
   }
 
-  function translateCountries() {
-    if (
-      location?.country?.value &&
-      !shipsTo.includes(location.country.value as string)
-    ) {
-      shipsTo.push(location.country.value as string)
-    }
```
Comment (polishq) [inline react/components/LocationForm.tsx:589]: Why was this removed?

File: react/components/LocationForm.tsx [hunk 2]
```diff
@@ -580,25 +583,36 @@ const LocationForm: FunctionComponent<WrappedComponentProps & AddressProps> = (
...
+  useEffect(() => {
+    const translateCountries = () => {
```
[the remark this answers is not in the document]
Comment (Arturo777) [reply react/components/LocationForm.tsx:587]: Hi @arturmagalhaesjr  `shipsTo` variable doesn't get updated during the user interaction, and however i think that it's better to declare a function inside the useEffect if that function is only used there.
